sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue/AsyncSamples.java

```diff
@@ -20,7 +20,7 @@ public class AsyncSamples {
      */
     public static void main(String[] args) {
         // Create an async queue client.
-        String queueURL = String.format("https://%s.queue.core.windows.net/%s%s", ACCOUNT_NAME, QUEUE_NAME, SAS_TOKEN);
+        String queueURL = String.format("https://%s.queue.core.windows.net/?%s%s", ACCOUNT_NAME, QUEUE_NAME, SAS_TOKEN);
```

Comment:
I think the query parameter start is in the wrong spot, shouldn't it be after the queue name as that is part of the URL path?

```suggestion
        String queueURL = String.format("https://%s.queue.core.windows.net/%s?%s", ACCOUNT_NAME, QUEUE_NAME, SAS_TOKEN);
```